src/timely-util/src/containers/array.rs

```diff
@@ -120,6 +128,22 @@ impl<T> Deref for Array<T> {
     }
 }
 
+impl<T> DerefMut for Array<T> {
+    fn deref_mut(&mut self) -> &mut Self::Target {
+        // TODO: Use `slice_assume_init_ref` once stable.
```

Comment:
Probably `slice_assume_init_mut` rather than `_ref`, I think?

Reply:
If nothing else, we'll want to have the `unsafe` justification for `_mut` instead, I think. Maybe a nit, but "have the right explanation when we use unsafe" seems like a great place to pick nits.

Reply:
Code has been removed.